mlflow/server/js/src/components/ExperimentView.js

```diff
@@ -115,7 +115,7 @@ class ExperimentView extends Component {
       } else if (sort.key === 'user_id') {
         sortValue = Utils.formatUser(runInfo.user_id);
       } else if (sort.key === 'source') {
-        sortValue = Utils.baseName(runInfo.source_name);
+        sortValue = Utils.renderSource(runInfo);
```

Comment:
Fixed sorting as well; tested in UI (see screenshot in description).

Reply:
We should make sure that renderSource returns a string here, otherwise things will be messed up if we ever make it return an HTML element (as some of our other render methods do). Maybe we should call it formatSource instead.

Reply:
Actually even renderSource might return an element if it wants to insert a link to GitHub there.

Reply:
Good point. I will undo this, and let the sorting be exactly as weird as before.